src/sentry/monitors/models.py

```diff
@@ -341,6 +341,8 @@ def get_alert_rule_data(self):
                     )
             alert_rule_data["targets"] = targets
 
+            alert_rule_data["environment_id"] = alert_rule.environment_id
```

Comment:
we need the environment name, not ID. Let's keep this consistent with what we've serialized

Reply:
hmmm...let me see what existing rules look like but pretty sure it just returns the id

Reply:
I'm saying for our API, we POST / PUT with the name, so in our response we also want to return the name right?

Reply:
ah...it uses a different serializer...

Reply:
I think this can just become `environment__name`

Reply:
using `environment`

Reply:
only the `id` is stored not an fk